kafka-plugins-0.10/widgets/Kafka-batchsource.json

```diff
@@ -5,12 +5,31 @@
   "display-name": "Kafka Consumer",
   "configuration-groups": [
     {
-      "label": "Kafka Configuration",
+      "label": "Connection",
       "properties": [
         {
-          "widget-type": "textbox",
-          "label": "Reference Name",
-          "name": "referenceName"
+          "widget-type": "toggle",
+          "label": "Use Connection",
+          "name": "useConnection",
+          "widget-attributes" : {
+            "on": {
+              "value": "true",
+              "label": "YES"
+            },
+            "off": {
+              "value": "false",
+              "label": "NO"
+            },
+            "default": "false"
+          }
+        },
+        {
+          "widget-type": "connection-select",
+          "label": "Connection",
+          "name": "connection",
+          "widget-attributes": {
+            "placeholder": "connection id"
```

Comment:
Does this select dropdown list all of the connectors? Shouldn't this widget have some way to specify the connector type?

Reply:
Changed this to `connectionType` and removed placeholder after talked with Edwin